integration-test/src/main/java/com/sequenceiq/it/config/IntegrationTestConfiguration.java

```diff
@@ -31,7 +31,7 @@
 import io.jaegertracing.spi.Sampler;
 
 @Configuration
-@ComponentScan("com.sequenceiq.it")
+@ComponentScan(basePackages = { "com.sequenceiq.it", "com.sequenceiq.cloudbreak.cloud.gcp.util" })
```

Comment:
Why do we need this package for integration test module?

Reply:
because now we have a bean and not a static class